components/package-template/src/etc/clp-config.template.json.yaml

```diff
@@ -3,6 +3,15 @@
 #  storage_engine: "clp-s"
 #  query_engine: "clp-s"
 #
+## API server config
+#api_server:
+#  host: "localhost"
+#  port: 3001
+#  default_max_num_query_results: 1000
+#  query_job_polling:
+#    initial_backoff_ms: 100
+#    max_backoff_ms: 5000
+#
```

Comment:
can't comment on the `package:` line:

Let's leave the `package:` and `api_server:` blocks uncommented

Reply:
and follow the formatting suggestions in the text template - leave one empty line after the server syntax hint file line